celery/backends/redis.py

```diff
@@ -340,3 +341,22 @@ def db(self):
     @deprecated.Property(4.0, 5.0)
     def password(self):
         return self.connparams['password']
+
+
+class SentinelBackend(RedisBackend):
+    """Sentinel task result store."""
+
+    def __init__(self, *args, **kwargs):
+        super(SentinelBackend, self).__init__(*args, **kwargs)
+
+        _get = self.app.conf.get
+
+        self.broker_url = _get('BROKER_URL') or {}
```

Comment:
This module contains the implementation of the result backend, so my question is why are the settings for the broker used here?